modules/swagger-servlet/src/test/java/io/swagger/servlet/resources/ApiOperationResource.java

```diff
@@ -0,0 +1,9 @@
+package io.swagger.servlet.resources;
+
+public class ApiOperationResource {
+
+    @Uber(description = "Description")
+    public String getResponse() {
+        return "Response";
+    }
+}
```

Comment:
test should be part of swagger-jaxrs, not swagger-servlet (or even better have them both)

Reply:
Test added to JaxRS module